curation_concerns.gemspec

```diff
@@ -45,6 +45,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'dry-struct', '~> 0.1'
   spec.add_dependency 'flot-rails', '~> 0.0.7'
   spec.add_dependency 'highcharts-rails'
+  spec.add_dependency 'hydra-pcdm', '~> 0.9.0'
```

Comment:
Merged, @escowles. Thanks! HW now needs a release and this PR can then point at the new release.